mars/services/storage/transfer.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2020 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+from dataclasses import dataclass
+from typing import Union
+
+from ... import oscar as mo
+from ...config import options
+from ...storage import StorageLevel
+from ...utils import dataslots, extensible
+from .core import StorageManagerActor, StorageHandlerActor
+
+
+@dataslots
+@dataclass
+class TransferMessage:
```

Comment:
We may only serialize data classes with pickle, which is of low efficiency with large blocks. Try using Serializables instead.